docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -47,20 +47,21 @@ Kaskada is built on the idea of a _timeline_ - the history of how a value change
 ]
 ....
 
-Timelines give you declarative queries and the ability to reason about temporal context, time travel, sequencing, time-series, and more.
-Transforming and combining timelines allow you to intuitively express computations over events.
+Timelines allow you to reason about temporal context, time travel, sequencing, time-series, and more. 
+They allow simple, composable, declarative queries over events. 
+Transforming and combining timelines allows you to intuitively express computations over events.
 
 ****
 ⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
 ****
 
-With Kaskada, it's timelines all the way down - every operation starts from timelines and produces timelines. 
-When you make a query, the resulting timeline is interpreted in one of two ways: as a _history_ or as a _snapshot_. 
+With Kaskada, it's timelines all the way down - every operation's inputs and outputs are timelines. 
+When you make a query, the resulting timeline is output in one of two ways: as a _history_ or as a _snapshot_. 
```

Comment:
This is the part that makes me skeptical. 

I think we went with a timeline is either discrete or continuous. Timeline is the abstraction. So the abstraction has 2 interpretations (I am avoiding the word "type" here). 

Getting the values *out of the timeline* and into something else (like a stream, a table, a data format) then has to make a decision on whether they would like to extract a snapshot or historical 

The way it is phrased here it allows for the reader to reach the conclusion that the result of a query is either historical or snapshot when earlier on we said it's all timelines (inputs and outputs). 

I think we are driving the point that 

1. Once your data is in the "system" it's timelines that you need to think of to operate on your data. 
2. Once you have the timeline that you want now you need to think on how to get the data out of the timeline and the form that you need them (stream, table, file etc.) Now you need to make a decision as to whether you want a snapshot from the timeline or the historical values from the timeline. 


But I am still digesting the names and their meaning.

Reply:
I think I agree. Previously, we've emphasized something about separating the `query` or `feature definition` or something (the formula or definition of the value to compute) and then `query parameters`. Basically, a way to make it clear that we separate "what to compute" from "how to compute it / output it / etc.".

I'm not sure we need to go that far in this getting started page -- although it may be helpful. But, I was trying to get at it some above -- in terms of saying "1. The result of a query is a timeline. 2. The timeline may be output as a history or a snapshot. This determines the data or content that is output. 3. The timeline history or timeline snapshot may be written to a stream (Kafka, Pulsar, etc.) or a table (Parquet, CSV, C*, etc.)".

We could maybe go a little further and try to clarify that the Fenl query is only the first step -- produces timelines. And the 2nd and 3rd step are "late bound" when you execute the query.

Reply:
See what you guys think about the revised wording.

I agree with Theo that we should make it more clear that `query -> timeline` in all cases. I tried to focus the "historical vs snapshot" distinction in the different use cases people might have.

So:
- Queries _output_ timelines (to tie back to operations inputs & outputs being timelines)
- Timelines are _used_ in different ways (relate to what a user wants to do with the timelin)
- Histories and Snapshots are _written_ as streams or tables (reflecting the mechanics of serialization)